Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/BoundingBox/BoundingBox.cs

```diff
@@ -649,109 +749,94 @@ public bool ShowRotationHandleForZ
         [SerializeField]
         [Tooltip("Check to draw a tether point from the handles to the hand when manipulating.")]
         private bool drawTetherWhenManipulating = true;
-        public bool DrawTetherWhenManipulating
-        {
-            get { return drawTetherWhenManipulating; }
-            set { drawTetherWhenManipulating = value; }
-        }
+
+        /// <summary>
+        /// Check to draw a tether point from the handles to the hand when manipulating.
+        /// </summary>
+        public bool DrawTetherWhenManipulating => drawTetherWhenManipulating;
```

Comment:
This (and quite a few other properties below) had their setters removed in this PR. Was this intentional?

Reply:
This also means that DrawTetherWhenManipulating will not be set-table from code, violates principle "make stuff configurable from code"

Reply:
no it wasn't....I didn't realize it only created getters and not setters